sdk/identity/identity-broker/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
```

Comment:
Missing Features Added section: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity.Broker/CHANGELOG.md

Reply:
done